### PR TITLE
ES|QL Add full-text search to the functions docs page

### PR DESCRIPTION
**Pull request by ChrisHegarty**
Now that the `match` and `qstr` functions are Tech Previewing we should add them to the top-level functions doc page.

Screenshot 2024-11-01 at 11 03 02

### CODE REVIEW

**Comment by ChrisHegarty** [comment]
we'll need to update the limitations page too:

https://github.com/elastic/elasticsearch/blob/main/docs/reference/esql/esql-limitations.asciidoc#full-text-search-is-not-supported

**Review by afoucret — APPROVED** [review]
Overall OK but I would add a preview flag for these function as done for others tech preview functions:

Capture d’écran 2024-11-01 à 12 00 22

**Comment by ChrisHegarty** [comment]
> Overall OK but I would add a preview flag for these function as done for others tech preview functions:

Thanks. Done.

Screenshot 2024-11-01 at 11 03 12

File: docs/reference/esql/esql-limitations.asciidoc [hunk 1]
```diff
@@ -102,11 +102,19 @@ is currently experimental.
 
 [discrete]
 [[esql-limitations-full-text-search]]
-=== Full-text search is not supported
+=== Full-text search
```
Comment (ioanatia) [inline docs/reference/esql/esql-limitations.asciidoc:105]: 🎉 🎉 🎉 !

**Review by ioanatia — APPROVED** [review]
nice!

@afoucret looks like we have a preview flag already?

Screenshot 2024-11-01 at 12 05 08

and in function docs we have a disclaimer
Screenshot 2024-11-01 at 12 05 35

unless you meant something else?

EDIT: Chris is too fast 😄

**Comment by ChrisHegarty** [comment]
> @afoucret looks like we have a preview flag already?

It's there now. @afoucret may have been looking at a slightly outdated preview or something. :-)

**Comment by afoucret** [comment]
My bad, I don't know why I thought it was missing

**Pull request merged**